pyproject.toml

```diff
@@ -23,8 +23,8 @@ license-files = ["LICEN[CS]E*"]
 dependencies = [
     "authlib>=0.15.0",
     "cryptography>=2.6",
-    "django>=2.2.0",
-    "djangorestframework>=3.11.0",
+    "django>=4.2.0",
+    "djangorestframework>=3.14.0",
```

Comment:
If I'm not completely mistaken 3.14.0 does not support Django 4.2 yet, only from 3.15.0 Django 4.2 is supported

Reply:
yeah, fixed it and updates test. Also updated tests to reflect that django 4.2 does not support python 3.13